### PR TITLE
feat: shortcuts modal

### LINKED ISSUE #49: Keyboard shorcut reference
We need a modal that can display all the active shortcuts. This modal should be displayed by clicking a button at the bottom-right of the editor viewport.

Depends on #17

### PR DESCRIPTION
**Pull request by nicosantangelo**
screen shot 2019-01-31 at 16 08 25

Closes #49

### CODE REVIEW

File: src/components/Modals/ShortcutsModal/ShortcutsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    return (
+      <Modal
+        open={modal.open}
+        className="ShortcutsModal"
+        size="small"
+        onClose={this.handleOnClose}
+        closeIcon={<CloseModalIcon onClick={this.handleOnClose} />}
+      >
+        <Modal.Content>
+          <div className="title">Shortcuts</div>
```
Comment (cazala) [inline src/components/Modals/ShortcutsModal/ShortcutsModal.tsx]: This component is missing i18n

File: src/components/Modals/ShortcutsModal/ShortcutsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+          <div className="shortcut-list">
+            <div className="subtitle">BASIC CONTROLS</div>
+
+            <div className="shortcuts">
+              <div className="shortcut">
+                <div className="name">Undo</div>
+                <div className="keybinding">
+                  <Chip text="⌘" />
```
Comment (belohlavek) [inline src/components/Modals/ShortcutsModal/ShortcutsModal.tsx]: How about ctrl for windows/linux?

Comment (nicosantangelo) [reply src/components/Modals/ShortcutsModal/ShortcutsModal.tsx]: Good question, should we ask design?

File: src/components/Modals/ShortcutsModal/ShortcutsModal.tsx (hunk 1 above)
Comment (nicosantangelo) [reply src/components/Modals/ShortcutsModal/ShortcutsModal.tsx]: Good catch!

**Pull request merged**